pkg/controllers/dynamicbindcontroller/dynamicbindcontroller.go

```diff
@@ -168,7 +168,7 @@ func (d *DynamicBindController) reconcileDynamicBindRequest(dynamicBindRequest *
 	for _, dynamicBindConfiguration := range dynamicBindConfigurations.Items {
 		environment, err := d.hfClientSet.HobbyfarmV1().Environments().Get(dynamicBindConfiguration.Spec.Environment, metav1.GetOptions{})
 
-		if provisionMethod, ok := environment.Annotations["provisioner"]; ok {
+		if provisionMethod, ok := environment.Annotations["hobbyfarm.io/provisioner"]; ok {
```

Comment:
Part of me says we should make this a constant and reference the constant here but you know what, that can just be tech debt :)